test/DebuggerCommon/parentedDynamicCode3.js.dbg.baseline

```diff
@@ -7,24 +7,24 @@
   {
     "sources": [
       {
-        "scriptId": 1,
         "fileName": "parentedDynamicCode3.js",
         "lineCount": 18,
-        "sourceLength": 750
+        "sourceLength": 750,
+        "scriptId": 1
       },
       {
-        "scriptId": 2,
-        "parentScriptId": 1,
         "scriptType": "eval code",
         "lineCount": 1,
-        "sourceLength": 11
+        "sourceLength": 11,
+        "scriptId": 2,
+        "parentScriptId": 1
       },
       {
-        "scriptId": 4,
-        "parentScriptId": 1,
         "scriptType": "eval code",
         "lineCount": 1,
-        "sourceLength": 11
+        "sourceLength": 11,
+        "scriptId": 5,

```

Comment:
why this change? Is it due to ChakraLib.js?

Reply:
Due to JsBuiltIn.js